extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -210,6 +231,10 @@ export class AmpA4A extends AMP.BaseElement {
         this.preconnect.url(p, true);
       });
     }
+    // TODO(tdrl): Temporary, while we're verifying whether SafeFrame is an
+    // acceptable solution to the 'Safari on iOS doesn't fetch iframe src from
+    // cache' issue.  See https://github.com/ampproject/amphtml/issues/5614
+    this.preconnect.preload(SAFEFRAME_IMPL_PATH);
```

Comment:
Not sure if we want preload here vs url (connect vs fetch) especially given that this is set for both control and experiment.  I recommend we start conservative with url especially given preload is only supported in chrome (http://caniuse.com/#search=preload) and we may elect to stick with the cache method where possible.